MAUI/Masked-Entry/Basic-Features.md

```diff
@@ -0,0 +1,201 @@
+
+## Setting CursorPosition
+
+Cursor position in the masked entry can either be obtained or updated using `CursorPosition` property in SfMaskedEntry.
+
+## Setting FontSize
+The Masked Entry control provides the user to customize the font size of the text inside the entry part using `FontSize` property of type double.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000" Value="1234"
+                          FontSize="18"/>
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.Value = 1234;
+maskedEntry.FontSize = 18;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontSize](MaskedEntry_Images/MaskedEntry_FontSize.png)
+
+## Setting FontAttributes
+
+The Masked Entry control provides the user to customize the font attribute of the text inside the entry part using `FontAttributes` property of type FontAttributes.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000" Value="1234"
+                          FontAttributes="Bold"/>
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.Value = 1234;
+maskedEntry.FontAttributes = FontAttributes.Bold;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontAttributes](MaskedEntry_Images/MaskedEntry_FontAttributes.png)
+
+## Setting FontFamily
+
+The Masked Entry control provides the user to customize the font family of the text inside the entry part using `FontFamily` property of type string.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000" Value="1234"
+                          FontFamily="Lobster-Regular"/>
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.Value = 1234;
+maskedEntry.FontFamily = "Lobster-Regular";  
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontFamily](MaskedEntry_Images/MaskedEntry_FontFamily.png)
+
+## Setting Keyboard
+
+The Masked Entry control provides the user to specifies the virtual keyboard that's displayed when entering text using `Keyboard` property of type Keyboard.
+
+
+## Setting Placeholder
+
+The Masked Entry control provides the user to defines the text that's displayed when the Mask and Value is empty using `Placeholder` property of type `string.` The default value of the Placeholder property is string.Empty.
```

Comment:
The Masked Entry control allows the user to define the text displayed when the Mask and Value are empty using the `Placeholder` property of type `string.` The default value of the Placeholder property is string.Empty.

Reply:
Addressed the changes.